types/combined-stream/index.d.ts

```diff
@@ -1,19 +1,28 @@
 // Type definitions for combined-stream 1.0
 // Project: https://github.com/felixge/node-combined-stream
-// Definitions by: Felix Geisendörfer <https://github.com/felixge>, Tomek Łaziuk <https://github.com/tlaziuk>
+// Definitions by: Felix Geisendörfer <https://github.com/felixge>, Tomek Łaziuk <https://github.com/tlaziuk>, Kon Pik <https://github.com/konpikwastaken>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 /// <reference types="node" />
 
-import { Stream } from "stream";
+import { Stream } from 'stream';
 
-declare class CombinedStream extends Stream implements CombinedStream.Options {
+type Appendable = NodeJS.ReadableStream | NodeJS.WritableStream | Buffer | string | NextFunction;
+// tslint:disable-next-line:no-any-union
+type NextFunction = (next: (stream: Appendable) => any | undefined) => any;
```

Comment:
```suggestion
type NextFunction = (next: (stream: Appendable) => any) => any;
```
`any` should include `undefined`. Is there a reason to include it explicitly?

Reply:
Resolved